Secrets.md

```diff
@@ -37,7 +37,7 @@ sekret {
         propertiesFile.set(File("custom/path/to/file.properties")) // default: sekret.properties in the current module folder
         
         nativeCopy {
-            // set output directories here
+            androidJNIFolder.set(file("custom/path/to/src/main/jniLibs")) // default does not copy the native libraries
```

Comment:
The default right now is the following:

It checks if `src/androidMain/jniLibs` exists and if not it checks if `src/main/jniLibs` exists.
It takes the first existing folder, so it does not need to be specified as long as the jniLibs folder exists in one of these targets.
This example uses application secrets without specifing the path: https://github.com/DatL4g/Mimasu/blob/v1/composeApp/build.gradle.kts#L250-L257

But we can add some kind of documentation of course!

I am not sure myself how a good explaination would look like, maybe something simple like this:
`default: src/androidMain/jniLibs or src/main/jniLibs`